content/doc/book/system-administration/viewing-logs.adoc

```diff
@@ -33,22 +33,30 @@ run `systemctl edit jenkins` and add the following:
 Environment="JENKINS_LOG=%L/jenkins/jenkins.log"
 ----
 
-=== Windows
+=== Windows (msi)
 
 By default logs should be at `%JENKINS_HOME%/jenkins.out` and
 `%JENKINS_HOME%/jenkins.err`, unless customized in
 `%JENKINS_HOME%/jenkins.xml`
```

Comment:
```suggestion
By default logs should be at `%JENKINS_HOME%/jenkins.out` and `%JENKINS_HOME%/jenkins.err`, unless customized in `%JENKINS_HOME%/jenkins.xml`
```
sentence per line formatting

Reply:
can you also please suggest what should be the content for other systems